auth0/v3/test/management/test_users_by_email.py

```diff
@@ -0,0 +1,37 @@
+import unittest
+import mock
+from ...management.users_by_email import UsersByEmail
+
+
+class TestUsersByEmail(unittest.TestCase):
+
+    @mock.patch('auth0.v3.management.users_by_email.RestClient')
+    def test_search_users_by_email(self, mock_rc):
+        mock_instance = mock_rc.return_value
+   
+        u = UsersByEmail(domain='domain', token='jwttoken')
+        u.search_users_by_email('A@B.com')
+
+        args, kwargs = mock_instance.get.call_args
+
+        self.assertEqual('https://domain/api/v2/users-by-email', args[0])
+        self.assertEqual(kwargs['params'], {
+            'email': 'a@b.com',
+            'fields': None,
```

Comment:
Does this mean that there's a `field=null` parameter being sent on the body?

Reply:
Yes we set this parameter to None before sending it, if the field array is empty on [this line]( https://github.com/saltukalakus/auth0-python/blob/58c016385610e4eb8c71e82788d287752b3cd6b0/auth0/v3/management/users_by_email.py#L41).